### PR TITLE
Add new methos Num

### LINKED ISSUE #4: BU men uchun .

### PR DESCRIPTION
**Pull request by rustamkulov**
Closes #4

### CODE REVIEW

File: DemoConsole/MuhammadaliDemo.cs [hunk 1]
```diff
@@ -6,4 +6,9 @@ public string Sigap(string text)
     {
         return text + "Sigap !";
     }
+
+    public int Num(int n)
+    {
+        return n;
+    }
```
Comment (ahadulla) [inline DemoConsole/MuhammadaliDemo.cs:13]: E bu bo'maydieeeeeeeeeeeeeeeeeeeeeeeeeeeeee

File: DemoConsole/MuhammadaliDemo.cs [hunk 2]
```diff
@@ -6,4 +6,9 @@ public string Sigap(string text)
     {
         return text + "Sigap !";
     }
+
+    public int Num(int n)
```
Comment (ubaydullayevboburjon) [inline DemoConsole/MuhammadaliDemo.cs:10]: E bomapti bu to'g'irlang eeee buni

File: DemoConsole/MuhammadaliDemo.cs [hunk 3]
```diff
@@ -6,4 +6,9 @@ public string Sigap(string text)
...
+    {
+        return n;
+    }
 }
```
Comment (ahadulla) [inline DemoConsole/MuhammadaliDemo.cs:14]: salomcha

File: DemoConsole/MuhammadaliDemo.cs [hunk 4]
```diff
@@ -6,4 +6,9 @@ public string Sigap(string text)
     {
         return text + "Sigap !";
```
Comment (ubaydullayevboburjon) [inline DemoConsole/MuhammadaliDemo.cs:7]: Nima gap

**Review by ubaydullayevboburjon — CHANGES_REQUESTED** [review]

**Review by ahadulla — CHANGES_REQUESTED** [review]

**Pull request merged**